vagrant-service-manager.spec

```diff
@@ -85,8 +85,10 @@ popd
 %{vagrant_plugin_instdir}/TODO
 
 %changelog
-* Thu Feb 25 2016 Navid Shaikh - 0.0.3-1
+* Thu Mar 01 2016 Navid Shaikh - 0.0.3-1
```

Comment:
I think Mar is correct.
Check <https://fedoraproject.org/wiki/Packaging:Guidelines#Multiple_Changelog_Entries_per_Release>